fitwsarah_frontend/src/views/PersonalTrainerPanel/TrainerFeedback.js

```diff
@@ -12,21 +13,57 @@ function Feedbacks() {
 
     const {
         isAuthenticated,
+        getAccessTokenSilently,
     } = useAuth0();
 
 
     const [feedbacks, setFeedbacks] = useState([]);
     const { t } = useTranslation('adminPanel');
+    const [accessToken, setAccessToken] = useState(null);
+
+        useEffect(() => {
+        if (isAuthenticated) {
+          const getAccessToken = async () => {
+            try {
+              const token = await getAccessTokenSilently({
+                audience: process.env.REACT_APP_AUTH0_AUDIENCE,
+                scope: configData.scope,
+              });
+              setAccessToken(token);
+            } catch (e) {
+              console.error(e.message);
+            }
+          };
+          getAccessToken();
+        }
+      }, [getAccessTokenSilently, isAuthenticated]);
+    
+        const fetchData = async () => {
```

Comment:
Could you elaborate on why you used the fetchData() right here?

Reply:
It is to fetch the accessToken that will be used to make API requests.